src/video_core/renderer_opengl/renderer_opengl.h

```diff
@@ -59,6 +60,7 @@ class RendererOpenGL : public RendererBase {
     void DrawScreens();
     void DrawSingleScreenRotated(const ScreenInfo& screen_info, float x, float y, float w, float h);
     void UpdateFramerate();
+    void FrameLimiter(Common::Profiling::Duration average_frame_time, int frame_limit);
```

Comment:
i need to find a way to get the last frame time, and all profiler classes represents them as Common::Profiler::Duration . it would be nice if i could  only just pass (float , int ) to FrameLimiter.
i could create a function in profiler_reporting.h that returns a float from Duration(already one in GetDataForColumn(), then use it in a parameter for FrameLimiter. then to prevent repititions, remove all duplications of  duration_to_float(). that is a lot of changes.